.eslintrc.js

```diff
@@ -11,10 +11,26 @@ module.exports = {
   ignorePatterns: ['*.d.ts', '**/proto/**/*.*'],
   rules: {
     // 2 -> error | 1 -> warning | 0 -> disabled
-    'no-console': 2,
+    'no-console': 1,
     'prettier/prettier': 2,
     // Disabling violated rules for JS compatibility
     '@typescript-eslint/no-explicit-any': 0,
-    '@typescript-eslint/explicit-module-boundary-types': 0
+    '@typescript-eslint/explicit-module-boundary-types': 0,
+    // errors after upgrading eslint to 8.3
+    // FIXME investigate each and fix instead of down-prioritize here
```

Comment:
shall we create an item to fix these?

Reply:
https://github.com/lightbend/kalix-javascript-sdk/issues/455